src/_hydra.pyx

```diff
@@ -290,10 +296,12 @@ cdef class BloomFilter:
         return min(len(BloomCalculations.PROBS) - 1, int(v))
 
     @classmethod
-    def _bucketsFor(cls, numElements, bucketsPer, filename, read_only, want_lock=False):
+    def _bucketsFor(cls, numElements, bucketsPer, filename, read_only, want_lock=False, fdatasync_on_close=True):
```

Comment:
minor nit - the argument lists for methods is getting long for my taste, but we can address this with a later refactor.